source/server/admin/prometheus_stats.cc

```diff
@@ -11,10 +11,8 @@ namespace Server {
 
 namespace {
 
+// TODO(mathetake) replace with re2 for safety.
```

Comment:
...and speed :)

Also note here that when you do that, you can make sanitizeName() take an absl::string_view and remove a std::string construction at one of the call-sites.

Reply:
> Also note here that when you do that, you can make sanitizeName() take an absl::string_view and remove a std::string construction at one of the call-sites.

oh sounds good.. thank you for letting me know that!

Reply:
The point was to put it in the TODO as it makes it possible to save another string-copy, which someone might easily forget to do when they switch RE2.